src/email/logs.contoller.js

```diff
@@ -4,8 +4,10 @@
     angular.module('motech-email')
         .controller('EmailLogsController', emailLogsController);
 
-    emailLogsController.$inject = ['$compile', '$scope', 'EmailLogsService', 'ModalWindow'];
-    function emailLogsController ($compile, $scope, EmailLogsService, ModalWindow) {
+    emailLogsController.$inject = ['$compile', '$scope', '$rootScope', 'EmailLogsService', 'ModalWindow'];
+    function emailLogsController ($compile, $scope, $rootScope, EmailLogsService, ModalWindow) {
+        $scope.isLoading = true;
+        $rootScope.$broadcast('motech-list.loading');
```

Comment:
Maybe use a more general event, and use $scope.$broadcast instead so it just effects related components